packages/calcite-components/src/components/button/button.scss

```diff
@@ -6,6 +6,10 @@
  * @prop --calcite-button-background-color: defines the component's background color.
  * @prop --calcite-button-border-color: defines the component's border color.
  * @prop --calcite-button-corner-radius: defines the component's corner radius.
+ * @prop --calcite-button-corner-radius-start-start: defines the component's start-start corner radius. Will fallback to --calcite-button-corner-radius if defined.
```

Comment:
Should this, and similar lines, say `if not defined` vs `if defined`?

Reply:
I was thinking about this. I'm not sure what makes more sense. It will only fallback to the more general `-corner-radius` var if that var is defined. Which is why I used `if defined` but maybe that's implied?

Reply:
> Are these just so Split Button can pass some through? Should these be made internal if so?

I thought about that. My reasoning for making them public was that customers may be using calcite-buttons to create their own components with similar edge case requirements so providing the option to set the corners separately would be beneficial